src/App.js

```diff
@@ -1,29 +1,22 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+/* eslint-disable */
+import Calculator from './components/Calculator';
+/* eslint-enable */
```

Comment:
## Error :red_circle:
![error](https://user-images.githubusercontent.com/93039359/184024425-2083b0fc-a964-465f-9830-3566fc2a8675.png)
![compnent](https://user-images.githubusercontent.com/93039359/184024445-9b063819-5bc0-48cb-89de-8a3ab6635051.png)

- [ ] You can see that I got an error because the component name's first letter is a small case letter. Please make it in the capital. :thinking:
- [ ] Always remember that if you make a component then its name's first letter always be capital :blush:

Reply:
Noted thanks, I really suffered with this error you know I even kept wondering why it was passing on my local machine and failing on Github.